src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,327 @@
+                                        {item.displayName}
+                                    </Text>
+                                </View>
+                            </View>
+                            <View style={[styles.peopleRowCell, styles.flex4]}>
+                                <Text style={[styles.textLabel, styles.colorMuted]}>
+                                    {item.login}
+                                </Text>
+                            </View>
+                        </>
+                    )
+                }
+                <View
+                    style={[
+                        styles.peopleRowCell,
+                        styles.peopleBadgesContainer,
+                        this.props.windowWidth < 1100 ? styles.peopleBadgesMobile : styles.peopleBadgesContainerDesktop,
+                    ]}
+                >
+                    {
+                        this.props.session.email === item.login && (
+                            <View>
+                                <View style={[styles.peopleBadge]}>
+                                    <Text style={[styles.peopleBadgeText]}>
+                                        Admin
+                                    </Text>
+                                </View>
+                            </View>
+                        )
+                    }
+                </View>
+            </View>
+        );
+    }
+
+    renderHeader() {
+        return (
+            <View style={[styles.peopleRow, styles.peopleHeaderRow]}>
+                <View style={[styles.peopleRowCell, styles.peopleCheckbox]}>
+                    <Checkbox
+                        isChecked={this.state.selectedEmployees.length === this.props.policy.employeeList.length}
+                        onPress={() => this.toggleUser('ALL')}
+                    />
+                </View>
+                <View style={[styles.peopleRowCell, styles.flex4]}>
+                    <Text style={[styles.textStrong]}>
+                        {this.props.translate('workspace.people.assignee')}
+                    </Text>
+                </View>
+                {
+                    this.props.windowWidth >= 1100 && (
+                        <View style={[styles.peopleRowCell, styles.flex4]}>
+                            <Text style={[styles.textStrong]}>
+                                {this.props.translate('common.email')}
+                            </Text>
+                        </View>
+                    )
+                }
+                <View style={[styles.peopleRowCell, styles.flex1]} />
+            </View>
+        );
+    }
+
+    render() {
+        let data = [];
+        if (this.props.policy.employeeList && this.props.policy.employeeList.length !== 0) {
+            data = _.filter(this.props.policy.employeeList, email => this.props.personalDetails[email])
+                .map(email => this.props.personalDetails[email]);
```

Comment:
There are a few ways to improve this none of them major but pointing them out anyways in case it helps..

1. `_.filter()` should always return at least an empty array so the initialization step with `let` is unnecessary
2. If we chain `.map()` after `_.filter` technically we are using the native array method and not underscore. It's more common to see use of [`_.chain()`](https://underscorejs.org/#chain) if you need to combine array methods
3. We can use `lodashGet()` to make sure we access the value or a default of an object
4. `_.filter` _should_ remove any undefined values by default (I think)

With all of that we'd end up with something like

```js
const data = _.chain(lodashGet(this.props, 'policy.employeeList', []))
    .map(email => this.props.personalDetails[email])
    .filter()   
    .value();
```

Reply:
Great job! I will go ahead and get more familiar with the `underscore` functions since I would have never thought of that `chain()` function. 

Ad 1. I had to initialize it since I have had the conditional statement there so I could not have initialized the `data` variable inside of the if block and then use it later because of incorrect scope.